src/Maker/MakeAuthenticator.php

```diff
@@ -403,11 +403,12 @@ public function configureDependencies(DependencyBuilder $dependencies, InputInte
     private function providerKeyTypeHint(): string
     {
         $reflectionMethod = new \ReflectionMethod(AbstractFormLoginAuthenticator::class, 'onAuthenticationSuccess');
-        $typeHint = (string) $reflectionMethod->getParameters()[2]->getType();
-        if ($typeHint) {
-            $typeHint .= ' ';
+        $type = $reflectionMethod->getParameters()[2]->getType();
+
+        if (!$type instanceof \ReflectionNamedType) {
```

Comment:
`getType()` returns `\ReflectionNamedType` | `null` since 7.1. As maker is not used in production, I opted to check `instanceOf` vs the more performant `null ===` for better DX.